{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,70 +1,63 @@
+1. Get the repository
 
-3. If you want to create a cluster with a different name, run:
-```
-   $ kind create cluster --name <alt-project-name>
-```
+       $ git clone git@github.com:Love-My-Delta-Inc/{{ cookiecutter.project_slug }}.git
+       $ cd {{ cookiecutter.project_slug }}
+
+2. Prepare the environment variables. Edit the `.envrc` file to work for your environment.
 
-## Initial Project Setup
+### Run the kubernetes cluster and the {{ cookiecutter.project_slug }} app to develop the code
 
-For the initial setup once the project has been generated, you will need to
-compile the pinned versions of the project's Python package dependencies. The
-following command will generate the `requirements/*.txt` files, and you should
-check these into the repo in the next step:
+1. Load the environment variables.
 
-    $ make compile
+2. Start the local kubernetes cluster:
 
-At this point, it's probably a good idea to commit this initial version of the project
-to version control:
+       $ kind create cluster --name {{ cookiecutter.project_slug }}
+    * If you want to create a cluster with a different name, run:
 
-    $ git init
-    $ git add .
-    $ git commit -m "Initial commit"
+          $ kind create cluster --name <alt-project-name>
 
-You can then use any of a number of available git repository sites such as Bitbucket,
-Gitlab, or GitHub to host the project; follow their instructions to push the repo to their
-systems.
+2. You can verify that it is running with
 
-Then you are ready to build the project for the first time. The following command
-will build the project so you will be ready to start developing. First execution
-of this command may take a few minutes to finish:
+       $ kubectl cluster-info
 
-    $ tilt up
+3. Verify the context with:
 
-It may take a little bit of time for all the services to start up, and it's possible for
-the first run to fail because of timing conflicts. If you do see messages indicating there
-were errors during the first run, stop all the containers using Ctrl-C, and then try it again.
+       $ kubectl config current-context
 
-## Contributor setup
+4. To spin up the app and all the services such as the database, use
+       $ tilt up
 
-For subsequent contributors (after the project has been through initial setup and
-pushed to a git repo), the `make compile` step should be skipped unless the express
-intention is to update the package dependencies.
+You are now ready to edit the code.
+The app will be automatically reloaded when its files change.
 
-To prepare your development environment you can use the following commands:
+To stop the local kubernetes cluster once you are done working:
+
+       $ kind delete cluster --name {{ cookiecutter.project_slug }}
+
+
+### Update dependencies
+
+To update the backend app dependencies, you must edit the `backend/requirements/*.txt` files.
+Once you have made your changes, you need to regenerate the `backend/requirements/*.txt` files you must run:
```

Comment:
That is incorrect; you must modify the `.in` files in the requirements directory. The `*.txt` files are generated once you run `make compile`.